compiler/rustc_hir_analysis/src/astconv/bounds.rs

```diff
@@ -194,8 +194,7 @@ impl<'tcx> dyn AstConv<'tcx> + '_ {
 
         self.add_bounds(
             param_ty,
-            ast_bounds.iter().filter(|bound| {
-                match filter {
+            ast_bounds.iter().filter(|bound| match filter {
```

Comment:
One of the match arms below includes a `let-chain` in it's body. I believe that rustfmt failed to format that match arm before support for `let-chains` was added resulting in a failure to format the entire match expressions.